lib/helper/Appium.js

```diff
@@ -121,9 +121,13 @@ class Appium extends WebdriverIO {
     }
 
     if (this.options.windowSize === 'maximize') {
-      this.browser.windowHandleMaximize(false);
-    }
-    if (this.options.windowSize && this.options.windowSize.indexOf('x') > 0) {
+      this.browser.execute('return [screen.width, screen.height]').then((res) => {
```

Comment:
Yes, if you set windowSize in config we think, that you plan to use browsers to test